bee-queue/arena
# Feat bootstrap: upgrade to v4.6.0

#### Changes Made
I'm interesting on using some components from this version, so the first step is upgrading this dependency
#### Potential Risks
We should have more functionalities

#### Test Plan
1. Go to examples
2. Run npm run start:bee or the other scripts
3. See that the current views are ok

#### Checklist
